bears/documentation/DocGrammarBear.py

```diff
@@ -0,0 +1,110 @@
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocBaseClass import (
+    DocBaseClass)
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+
+class DocGrammarBear(DocBaseClass, LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    REQUIREMENTS = {PipRequirement('language-check', '1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Documentation', 'Spelling', 'Grammar'}
+    CAN_FIX = {'Documentation', 'Spelling', 'Grammar'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('java') is None:
+            return 'java is not installed.'
+        else:
+            try:
+                from language_check import LanguageTool, correct
+                LanguageTool
+                correct
+                return True
+            except ImportError:  # pragma: no cover
+                return 'Please install the `language-check` pip package.'
+
+    def process_documentation(self,
+                              parsed,
+                              natural_language,
+                              languagetool_disable_rules):
+        """
+        This fixes the parsed documentation comment.
+
+        :param parsed:
+            Contains parsed documentation comment.
+        :param natural_language:
+            A locale representing the language you want to have checked.
+            Default is set to 'en-US'.
+        :param languagetool_disable_rules:
+            List of rules to disable checks for.
+        :return:
+            A tuple of fixed parsed documentation comment and warning_desc.
+        """
+        # Defer import so the check_prerequisites can be run without
+        # language_check being there.
+        from language_check import LanguageTool, correct
+
+        tool = LanguageTool(natural_language)
+        tool.disabled.update(languagetool_disable_rules)
+
+        metadata = iter(parsed)
+
+        new_metadata = []
+        for comment in metadata:
+            matches = tool.check(comment.desc)
+            new_desc = correct(comment.desc, matches)
+            new_metadata.append(comment._replace(desc=new_desc))
+
+        return (new_metadata,
+                'Documentation has invalid Grammar/Spelling')
+
+    def run(self, filename, file, language: str,
+            docstyle: str='default', natural_language: str='en-US',
+            languagetool_disable_rules: typed_list(str)=()):
+        """
+        Checks the main description and comments description of documentation
+        with LanguageTool.
```

Comment:
I'd like a little summary of what language tool can fix plus a link to the documentation.